ui/app/styles/helper-classes/spacing.scss

```diff
@@ -95,6 +95,10 @@
   margin-bottom: -$spacing-m;
 }
 
+.has-top-margin-xxs {
+  margin: $spacing-xxs 0;
```

Comment:
From the name I would think this would only impact the top margin, but it impacts top and bottom. I know this seems overkill but if you need to bottom margin at 0, can you make another helper for just bottom margin and make this one just top?

Reply:
sounds good!